lib/Transforms/DataCopy.cpp

```diff
@@ -1,4 +1,4 @@
-//===- DmaGeneration.cpp - DMA generation pass ------------------------ -*-===//
+//===- DataCopy.cpp - Explicit data copying optimization pass --------- -*-===//
```

Comment:
Please add "AffineDma" to this file name so that it makes more sense contextually. 'DataCopy' is ambiguous and this also only deals with the Affine dialect.

Reply:
The pass is no longer about generating DMA alone, and generating DMAs is not the default. The copies are done via DMA only if an additional flag is provided; otherwise, it's just point-wise copies (via loads and stores) -- the way things move for example b/w GPU shared memory and its global memory.

Unfortunately, there isn't a good short name for this optimization. AffineExplicitCopy and AffineDataCopy are the best I can think of now.

Reply:
Ah I see, thanks for the explanation. AffineDataCopy seems fine then.

Reply:
Done - changed to AffineDataCopyGeneration for now.